### DISCUSSION TITLE
Installation isn't clear

### DISCUSSION THREAD

**Discussion by ollyde** [alpinejs/alpine#3634] [4. General]
Using Svelte with TypeScript

It says 
"Now import Alpine into your bundle and initialize it like so:"

Here
https://alpinejs.dev/essentials/installation

What exactly does this mean? 

I included it within +layout.svelte but that doesn't work, for example.

**Comment by ekwoka** [comment]
There is no reason to use Alpine with Svelte. Svelte will already do all the things Alpine does and having them both will only cause issues.

Don't take this the wrong way, it's just some advice, but you might be moving a bit too fast in learning frontend dev. Take the time to learn how modules work.

**Reply by ollyde** [reply]
Can't use this library without.
https://devdojo.com/pines/docs/how-to-use

@ekwoka also don't agree, alpine adds localized data to elements without seeking the head each time.

**Reply by ekwoka** [reply]
Svelte already has localized data.

And that pack is inappropriate to use with Svelte.

Alpine is designed for use with pure server rendered applications. Svelte is client rendered.

Idk what you mean by "seeking the head" but svelte doesn't do that, and Alpine would be adding overhead, not removing it in this case, as svelte will already be wanting to handle the things you'd be passing to Alpine, and it's only going to cause issues, not solve them.

Mixing two separate systems and having them modify the same Dom without any awareness of the other is a debugging nightmare.

**Reply by ollyde** [reply]
@ekwoka we are using SvelteKit which is SSR (optionally) also what are you talking about? Alpine is a UI library.

Go to https://devdojo.com/pines and https://alpinejs.dev/ I think you maybe commenting on the wrong library here.

**Reply by ekwoka** [reply]
I know what Alpine is.

I contribute to it...

Alpine is built for pure SSR apps to add interactivity on the front. Like Laravel (the TALL stack mentioned on the pine thing) or Django or whatever.

Svelte is a UI library too, so it's ALREADY doing what you would use alpine for.

So why have two libraries fighting eachother to modify the same Dom?

You could even just use the components from that site modified for svelte.

And sveltekit doesn't have an option for pure SSR. It's still running it's JS in the browser.

Trying to do this is a REALLY bad idea. It's like trying to put react and svelte together.

**Reply by ollyde** [reply]
@ekwoka no worries, my bad, some linked me Alpine and Pines for Svelte :-/ I see now that they are not compatible.
